src/components/navigation/index.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import styles from './navigation.css';
+import classnames from 'classnames';
+import keymirror from 'keymirror';
+
+export const constants = keymirror({
+  'GRAPHIQL': null,
+  'QUERIES': null,
+  'MUTATIONS': null,
+  'CACHE': null
+});
+
+function setActiveItem(activeItem) {
+  return function getClasses(name) {
+    return classnames(styles.item, {
+      [styles.active]: activeItem === name
+    })
+  }
+}
+
+function Navigation(props) {
+  const { GRAPHIQL, QUERIES, MUTATIONS, CACHE } = constants;
+  const defaultActiveItem = QUERIES;
+  const [active, setActive] = useState(defaultActiveItem);
+  const getClasses = setActiveItem(active);
+  const { client: { numberOfQueries, numberOfMutations } } = props;
+
+  return (
+    <div className={styles.container}>
```

Comment:
I'd love for us to explore using [Reach UI](https://reacttraining.com/reach-ui/tabs/) for these tabs so we can have some nice accessibility by default setups for core interactions

Reply:
that is a great idea. i'll patch this and incorporate it